src/pybind/mgr/dashboard/HACKING.rst

```diff
@@ -518,3 +518,70 @@ The settings management implementation will make sure that if you change a
 setting value from the Python code you will see that change when accessing
 that setting from the CLI and vice-versa.
 
+
+How to deal with asynchronous tasks in the front-end?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+All executing and most recently finished asynchronous tasks are displayed on the
```

Comment:
Suggestion: "running" instead of "executing".

Reply:
"Executing" is used in many places (including the UI), are you suggesting to change it everywhere or only in the documentation?

Reply:
IMO "running tasks" is more fitting in this context, so maybe it's worth changing it everywhere.

Reply:
Changing "executing" to "running" is not that easy, it implies creating a new PR to change backend code that uses "executing" as name variables, dictionary keys, and message strings.
I don't think it's worth the effort.

Reply:
"running tasks" is less ambiguous than "executing tasks", IMHO. In the context of the provided screen shot though, I don't think anyone would get confused about "Executing", but in this text I'd definitely prefer to read "running tasks".

Reply:
@p-na does that mean that @ricardoasmarques should only rename in the documentation?

Reply:
Yes, I think @p-na is talking about the documentation here.
Abnd what about changing "EXECUTING" to "RUNNING" (just the title in the list of tasks) - that wouldn't be a big deal, would it?

Reply:
IMO we should use the same name everywhere. 

@callithea @p-na are you ok if we merge the "EXECUTING" label, and create a separate PR in the future to rename everything from "executing" to "running" (including code variables)?

Reply:
@ricardoasmarques: I'm fine with that, thank you!